minor note re consortium members and fixup of quickstart new node (#207)
ConsenSys/doc.goquorum: docs/concepts/security-framework.md

Consortiums come with risks such as accidental data exposure and potential liabilities that must be managed.
They require governance structures that fulfill the concerns of participants equally.

!!! note

>     In a consortium network, every member does not need to be a validator, and can remain as members. Each
>     member will still have a full copy of the chain data and has full ability to transact (private or public)
>     on the network. To prevent the network halting, all validator have additional responsibility to ensure
>     that their node does not go down and is available per SLAs of the consortium agreements.

Reviewer: ```suggestion
    In a consortium network, every member does not need to be a validator. Each non-validating
    member will still have a full copy of the chain data and has full ability to transact (private or public)
    on the network. To prevent the network halting, all validators have additional responsibility to ensure
    that their node does not go down and is available per SLAs of the consortium agreements.
```
